### PR TITLE
fix unbalanced quotes string

### PR DESCRIPTION
**Pull request by Chyroc**
# Pull Request for issue

#18 

## Description

* fix unbalanced quotes string
* speed up 100~150 ns/op

## Testing Instructions

image

### CODE REVIEW

File: pkg/util/strs.go [hunk 1]
```diff
@@ -44,57 +44,57 @@ func ToString(i interface{}) string {
...
-
 	return ret, nil
 }
 
-func appendIfBufferNotEmpty(buf *bytes.Buffer, list []string) []string {
-	if len(buf.String()) > 0 {
-		list = append(list, buf.String())
-		buf.Reset()
+func scanForByte(s string, pos int, r byte) (int, string, error) {
+	var ret []byte
```
Comment (kasvith) [inline pkg/util/strs.go]: byte buffer can increase perfomance

**Comment by Chyroc** [comment]
by test, bytes slice is faster

image

image

**Comment by kasvith** [comment]
can you also try with stringBuilder? it has a write byte too...

**Comment by Chyroc** [comment]
code:
image

result:

image
image

**Comment by kasvith** [comment]
Builder seems better for me

**Comment by Chyroc** [comment]
done

**Comment by kasvith** [comment]
tnx @Chyroc

**Pull request merged**